internal/lib/lib.go

```diff
@@ -1725,3 +1727,18 @@ func GetThrottle(key string) *int32 {
 	throttle := int32(throttle[key])
 	return &throttle
 }
+
+func UpdateV6(vip *avimodels.Vip, vipNetwork *akov1alpha1.AviInfraSettingVipNetwork) {
+	v6addr := strings.Split(vipNetwork.V6Cidr, "/")[0]
+	v6mask, _ := strconv.Atoi(strings.Split(vipNetwork.V6Cidr, "/")[1])
+
+	v6cidr := avimodels.IPAddrPrefix{
+		IPAddr: &avimodels.IPAddr{
+			Addr: &v6addr,
+			Type: proto.String("V6"),
+		},
+		Mask: proto.Int32(int32(v6mask)),
+	}
+	vip.AutoAllocateIPType = proto.String("V4_V6")
```

Comment:
Here We are assuming both `v4` and `v6` has been provided. Can user just specify `v6`?

Reply:
updated to allow v6 only